sdk/communication/azure-communication-phonenumbers/test/test_sip_routing_client_e2e.py

```diff
@@ -32,135 +36,119 @@ def test_get_trunks_from_managed_identity(self):
         self._sip_routing_client.set_routes(new_routes)
         result_routes = self._sip_routing_client.get_routes()
         assert result_routes is not None, "No routes were returned."
-        self._routes_are_equal(result_routes,new_routes)
+        routes_are_equal(result_routes,new_routes)
 
     def test_set_routes_from_managed_identity(self):
-        new_routes = [SipTrunkRoute(name="Alternative rule", description="Handle numbers starting with '+999'", number_pattern="\\+999[0-9]+", trunks=["sbs2.sipconfigtest.com"])]
+        new_routes = [SipTrunkRoute(name="Alternative rule", description="Handle numbers starting with '+999'", number_pattern="\\+999[0-9]+", trunks=[self.TRUNKS[1].fqdn])]
         client = self._get_sip_client_managed_identity()
         client.set_routes(self.ROUTES)
         client.set_routes(new_routes)
         result_routes = client.get_routes()
         assert result_routes is not None, "No routes were returned."
-        self._routes_are_equal(result_routes,new_routes)
+        routes_are_equal(result_routes,new_routes)
 
     def test_delete_trunk(self):
         trunk_to_delete = self.TRUNKS[1].fqdn
         self._sip_routing_client.delete_trunk(trunk_to_delete)
         new_trunks = self._sip_routing_client.get_trunks()
-        self._trunks_are_equal(new_trunks,[self.TRUNKS[0]])
+        trunks_are_equal(new_trunks,[self.TRUNKS[0]])
 
     def test_delete_trunk_from_managed_identity(self):
         trunk_to_delete = self.TRUNKS[1].fqdn
         client = self._get_sip_client_managed_identity()
         client.delete_trunk(trunk_to_delete)
         new_trunks = client.get_trunks()
-        self._trunks_are_equal(new_trunks,[self.TRUNKS[0]])
+        trunks_are_equal(new_trunks,[self.TRUNKS[0]])
 
     def test_add_trunk(self):
-        new_trunk = SipTrunk(fqdn="sbs3.sipconfigtest.com", sip_signaling_port=2222)
+        new_trunk = SipTrunk(fqdn=self.additionalTrunkFqdn, sip_signaling_port=2222)
         self._sip_routing_client.set_trunk(new_trunk)
         new_trunks = self._sip_routing_client.get_trunks()
-        self._trunks_are_equal(new_trunks,[self.TRUNKS[0],self.TRUNKS[1],new_trunk])
+        trunks_are_equal(new_trunks,[self.TRUNKS[0],self.TRUNKS[1],new_trunk])
 
     def test_add_trunk_from_managed_identity(self):
-        new_trunk = SipTrunk(fqdn="sbs3.sipconfigtest.com", sip_signaling_port=2222)
+        new_trunk = SipTrunk(fqdn=self.additionalTrunkFqdn, sip_signaling_port=2222)
         client = self._get_sip_client_managed_identity()
         client.set_trunk(new_trunk)
         new_trunks = client.get_trunks()
-        self._trunks_are_equal(new_trunks,[self.TRUNKS[0],self.TRUNKS[1],new_trunk])
+        trunks_are_equal(new_trunks,[self.TRUNKS[0],self.TRUNKS[1],new_trunk])
 
     def test_get_trunk(self):
-        trunk = self._sip_routing_client.get_trunk(self.TRUNKS[0].fqdn)
+        actual_trunks = self._sip_routing_client.get_trunks()
```

Comment:
For testcases, where we retrieve trunk by it's fqdn, we have to modify the domain, otherwise recorded tests would fail. (domain is generated with each test fixture run).